confd/templates/northbound/northbound.properties.tmpl

```diff
@@ -15,6 +15,8 @@ kafka.hosts={{ getv "/kilda_kafka_hosts" }}
 
 server.contextPath=/api
 
+web.request.asyncTimeout=30000
```

Comment:
Other NB options are starts with `northbound.` why this one differ?

Reply:
even from this file I could see a plenty of options that starts with something other than northbound prefix

Reply:
ok